src/Action.ts

```diff
@@ -13,6 +13,7 @@ export enum ActionTypes {
   SET_ENTITY = 'SET_ENTITY',
   DISPATCH = 'DISPATCH',
   CHANGE_MODEL = 'CHANGE_MODEL',
+  FLOW = 'FLOW',
```

Comment:
Note that FLOW has changed names.  It's now called Power Automate, so the term "FLOW" is going away.  Probably not a good idea to start with an obsolete name.  Might be good to try a generic name for this so we don't get caught up in rebrandings.   Maybe "workflow"?  Any other ideas?

Reply:
How about "EXTERNAL_WORKFLOW" or "REMOTE_WORKFLOW"?

Reply:
I like "REMOTE_WORKFLOW"

Reply:
We can always change the UI to display the current branding of FLOW/PA